server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/service/EventDeadLettersRedeliverTaskTest.java

```diff
@@ -64,6 +66,13 @@ void redeliverAllTaskShouldMatchJsonSerializationContract() throws Exception {
             .bean(TASK_ALL)
             .json(SERIALIZED_ALL)
             .verify();
+
+        EventDeadLettersRedeliverAllTask taskAllWithLimit = new EventDeadLettersRedeliverAllTask(SERVICE, new RunningOptions(Limit.limit(10)));
+
+        JsonSerializationVerifier.dtoModule(EventDeadLettersRedeliverAllTaskDTO.module(SERVICE))
+            .bean(taskAllWithLimit)
+            .json("{\"type\":\"event-dead-letters-redeliver-all\", \"runningOptions\":{\"limit\": 10}}")
```

Comment:
Can we still deserialize a JSON without runningOptions ?

Reply:
It will handle case old tasks already in the database, without runningOptions?

Reply:
I mean, do we have a test for this?